### PR TITLE
Re format copyright

### PR DESCRIPTION
**Pull request by gernotstarke**
removed text-centered,
added "box" to emphasize copyright

### CODE REVIEW

File: docs/00-preamble/copyright.adoc [hunk 1]
```diff
@@ -18,15 +17,22 @@ Lizenzverträge, die alles umfassend regeln, sind vorhanden.
 3.	Falls Sie weder unter die Kategorie 1. noch unter die Kategorie 2. fallen, aber dennoch die Dokumente und/oder Lehrpläne nutzen möchten, nehmen Sie bitte ebenfalls Kontakt unter info@isaqb.org zum iSAQB e. V. auf.
 Sie werden dort über die Möglichkeit des Erwerbs entsprechender Lizenzen im Rahmen der vorhandenen Lizenzverträge informiert und können die gewünschten Nutzungsgenehmigungen erhalten.
 
+.Wichtiger Hinweis
+****
 *Grundsätzlich weisen wir darauf hin, dass dieser Lehrplan urheberrechtlich geschützt ist.
 Alle Rechte an diesen Copyrights stehen ausschließlich dem International Software Architecture Qualification Board e. V. (iSAQB^(R)^ e. V.) zu.*
+****
+// end::DE[]
```
Comment (programming-wolf) [inline docs/00-preamble/copyright.adoc]: This is end-tag appears further down below, and will thus break/cause something to disappear. This tag has to be removed (as you did in the foundation repository)

**Review by programming-wolf — CHANGES_REQUESTED** [review]
That tag has to be removed, the rest looks nice.

File: docs/00-preamble/copyright.adoc (hunk 1 above)
Comment (gernotstarke) [reply docs/00-preamble/copyright.adoc]: right you are. Sorry, my mistake.

Comment (gernotstarke) [reply docs/00-preamble/copyright.adoc]: I added another commit - please re-consider merging - thx

**Review by programming-wolf — APPROVED** [review]
Thanks for fixing this. 🙂

**Pull request merged**